tests/test_buildah_authentication.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+# test_buildah_authentication
+# A script to be run at the command line with Buildah installed.  
```

Comment:
Extra whitespace at the end of this line.

Reply:
fixed